db/schema.rb

```diff
@@ -81,6 +81,14 @@
     t.index ["questionnaire_id"], name: "index_measurements_on_questionnaire_id", using: :btree
   end
 
+  create_table "one_time_responses", force: :cascade do |t|
+    t.string   "token",       null: false
+    t.integer  "protocol_id", null: false
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+    t.index ["protocol_id"], name: "index_one_time_responses_on_protocol_id", using: :btree
+  end
+
```

Comment:
Dit gaat volgens mij straks een conflict geven omdat dat nog van het andere pullrequest is.